app/assets/javascripts/controllers.js

```diff
@@ -17,6 +17,27 @@ Prometheus.Graph = {
     's': 1
   },
 
+  durationToString: function(seconds) {
+    // TODO(stuart): this is always returning seconds
+    // look to round to a certain amount based on the units
```

Comment:
Hmm, I can't parse this sentence somehow, or is a new one? (please use sentence-style caps and punctuation)